### ISSUE TITLE
Training on atomization or absolute energies?

### ISSUE THREAD

**Issue by proteneer** [isayev/ASE_ANI#12]
Another point that wasn't mentioned in the paper is if you guys were training to the total QM energy or the Atomization Energies (since you guys do provide the self-interaction atomic energies). Can you kindly clarify?

**Comment by Jussmith01** [comment]
Training is to atomization energies.

**Comment by proteneer** [comment]
Thanks Justin (for this and the other issue!)

**Comment by proteneer** [comment]
I need to clarify something else, are you guys normalizing the energies to the unit gaussian? We think Adrian may have mentioned something about this during the talk he gave at our company, but we couldn't find any references to this in the pre-print of the Chemical Science paper.

**Comment by Jussmith01** [comment]
Are you you asking: are we normalizing the labels [energies] (after we remove the single atom energies from the total energy)? If so, the answer is no. We actually do no preprocessing on the energies aside from the atomization shift. Once the energies are shifted we train directly to hartrees, which are in the range -4 to 0 Ha (Figure 3C -> https://arxiv.org/pdf/1708.04987v1.pdf) unless something is wrong with the data (or working with larger systems in the training set). Early on in this work I noticed normalizing didn't make a difference so I stopped using it. However, if working with larger molecules in the training set it might be a good idea to start doing it. Does this answer your question?
